### ISSUE TITLE
Import people blocked by CSP

### ISSUE THREAD

**Issue by garethbowen** [medic/medic#5675]
**Describe the bug**
Uploading people using the admin app isn't working.

**To Reproduce**
1. Create a JSON file with some people data in it. A minimal example:
  `[{"_id": "a", "name": "something"}]`
1. Go to Admin app > Import & Export > People
2. Click on the Browse button in the Import section
3. Select the file you created in 1.
4. Click Submit
5. An error is shown in the UI. There's an error in the log. The person is not imported.

**Expected behavior**
No errors are shown and a contact is imported.

**Logs**
```
Content Security Policy: The page’s settings blocked the loading of a resource at http://_db/undefined (“connect-src”).
```

**Screenshots**
![Screenshot from 2019-05-17 11-13-15](https://user-images.githubusercontent.com/2064938/57892882-d83ed480-7894-11e9-9539-505670264ebb.png)
![Screenshot from 2019-05-17 11-13-24](https://user-images.githubusercontent.com/2064938/57892883-d83ed480-7894-11e9-9447-37c27fe8dd41.png)

**Environment**
- Instance: localhost
- Browser: Firefox
- Client platform: Ubuntu
- App: admin
- Version: 3.5.0 (master)

**Additional context**
1. Before starting on this bug consider if we can drop this feature altogether. medic-conf provides similar functionality.
2. If it is worth fixing, consider refactoring it to use the existing APIs for importing people and places.
3. Also consider using bulk APIs instead of one call per contact.

**Comment by garethbowen** [comment]
I think this was caused by the URL changing here: #5361

**Comment by garethbowen** [comment]
I've fixed the bug (in a branch) and left all the improvements in a [separate issue](https://github.com/medic/medic/issues/5678) to help get 3.5.0 out the door.

**Comment by ngaruko** [comment]
Before
![image](https://user-images.githubusercontent.com/6979995/58057846-b312d480-7bba-11e9-8a7a-7c964aa017bf.png)

After
![image](https://user-images.githubusercontent.com/6979995/58060152-105f5380-7bc4-11e9-9a71-4ea3946c3879.png)

LGTM. Contacts imported. No errors in console. The UI does not show any indication of successful import and the imported file still shows. I guess it is not worth any more time since the feature is not even a popular use case anyhow. Your thoughts @garethbowen ?

**Comment by garethbowen** [comment]
Yeah I just went for the minimal change to get the page back to its original functionality. Let's see if it survives #5678 and if it's proven to be useful we can make the UI better.

**Comment by garethbowen** [comment]
Merged.
